templates/5_rename_and_reorganize.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # These two cells be re-run multiple times to combine different runs together
</pre><p>Move this note into its own bolded/enlarged markdown cell below, since this is pretty important.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>